chart/templates/pgbouncer/pgbouncer-deployment.yaml

```
@@ -66,6 +66,9 @@ spec:
       annotations:
         checksum/pgbouncer-config-secret: {{ include (print $.Template.BasePath "/secrets/pgbouncer-config-secret.yaml") . | sha256sum }}
         checksum/pgbouncer-certificates-secret: {{ include (print $.Template.BasePath "/secrets/pgbouncer-certificates-secret.yaml") . | sha256sum }}
+{{- if .Values.pgbouncer.extraAnnotations }}
+{{- toYaml .Values.pgbouncer.extraAnnotations | nindent 8 }}
+{{- end }}
```

```suggestion
      {{- if .Values.pgbouncer.extraAnnotations }}
        {{- toYaml .Values.pgbouncer.extraAnnotations | nindent 8 }}
      {{- end }}
```

Let's indent it like this instead.

---

Just to add a little more color here, there is nothing wrong with how you have it, we just had a few other PRs recently that did some formatting cleanup and I'd like to keep things consistent 👍